editor/components/provider/index.js

```diff
@@ -32,6 +32,20 @@ import store from '../../store';
  */
 const DEFAULT_SETTINGS = {
 	alignWide: false,
+	blockTypes: true,
```

Comment:
Same here, this ensure all blocks are available by default.

Reply:
See #4707. Includes a comment in that PR.

Reply:
I just merged, so let's drop the change here.